### ISSUE TITLE
A bug? two menu item hightlight at same time

### ISSUE THREAD

**Issue by Troy-Yang** [eddiemf/vue-scrollactive#21]
See below screenshot from online demo, You should recreate the issue with below steps:
1. Open demo page
2. Click on section 2
3. Scroll up a little to section 1.

Fix:
```
onScroll(event) {
        this.currentItem = this.getItemInsideWindow();
        if (this.currentItem !== this.lastActiveItem && this.currentItem) {
          this.removeActiveClass();
          this.$emit('itemchanged', event, this.currentItem, this.lastActiveItem);
          this.lastActiveItem = this.currentItem;
        }
        // Current item might be null if not inside any section
        if (this.currentItem) {
          this.removeActiveClass();
          this.currentItem.classList.add(this.activeClass);
        }
      },
```

I also added an empty check on line 3 because sometimes, i found the currentItem is undefined but the itemchanged even still triggered.

![screen shot 2018-05-21 at 6 24 31 pm](https://user-images.githubusercontent.com/3301669/40303157-43fdd776-5d24-11e8-95f7-e93c6a2d9e80.png)

**Comment by eddiemf** [comment]
Hello there,

Could you give me more information regarding OS and Browser version?
I'm testing on MacOS High Sierra 10.13.4 on Chrome version 66.0.3359.181 and looks like it's working as always.

**Comment by Troy-Yang** [comment]
Hey,

My OS is totally same as yours and chrome version is 66.0.3359.139. 

I tried in Safari and other machine chrome, both them works good normal. So, it only happened in my machine.

I did a debug and i found the difference: In safari, when I click the section links, it will trigger onScroll method when running at scrollTo => window.addEventListener('scroll', _this3.onScroll);. then it will update the currentItem instance and make the left thing works good. However, in my chrome, the onScroll method will not be triggered by window.addEventListener('scroll', _this3.onScroll);.
